library-app/src/components/pages/Login/Login.tsx

```diff
@@ -1,139 +1,120 @@
     }
-    if (!enteredEmailIsValid || !enteredPasswordIsValid) {
-      console.log('greska')
-    } else {
-      await LoginRequest(formData)
-        .then(({ data }) => {
-          localStorage.setItem('accessToken', data.accessToken)
-          localStorage.setItem('refreshToken', data.refreshToken)
-          localStorage.setItem('expiration', data.expiration)
-          navigateTo('/')
-          console.log(data)
-        })
-        .catch((error) => {
-          if (axios.isAxiosError(error)) {
-            if (error.response?.status === 401) {
-              setInvalidCredentials(true)
-              console.log('pogresni kredencijali')
-            }
-          }
-        })
-    }
-  }
-
-  const unmatchedCredentials = invalidCredentials
-  const unmatchedCredentialsClasses = unmatchedCredentials ? styles['invalid-credentials'] : ''
 
-  const emailInputIsInvalid = !enteredEmailIsValid
-  const emailInputClasses = emailInputIsInvalid
-    ? `${styles['form-group']} ${styles['invalid']}`
-    : styles['form-group']
-
-  const passwordInputIsInvalid = !enteredPasswordIsValid
-  const passwordInputClasses = passwordInputIsInvalid
-    ? `${styles['form-group']} ${styles['invalid']}`
-    : styles['form-group']
+    LoginRequest(formData)
+      .then(({ data }) => {
+        setIsLoggedIn(true)
+        setAccessToken(data.AccessToken)
+        setLocalStorage(data)
+        navigateTo('/')
+      })
+      .catch((error) => {
+        if (axios.isAxiosError(error) && error.response?.status === 401) {
+          setInvalidCredentials(true)
+        }
+      })
+  }
 
   return (
-    <MainLayout>
-      <div className={styles.container}>
-        <h1>Login</h1>
-        <form className={styles['form']} onSubmit={formSubmissionHandler}>
-          <div className={emailInputClasses}>
-            <label htmlFor='email'>
-              {emailInputIsInvalid ? 'Please enter email' : 'Email'}
-              {}
-            </label>
-            <input
-              type='email'
-              id='email'
-              value={enteredEmail}
-              onChange={handleEmailChange}
-              onBlur={handleOnBlurEmail}
-            />
-          </div>
-          <div className={passwordInputClasses}>
-            <label htmlFor='password'>
-              {passwordInputIsInvalid ? 'Please enter password' : 'Password'}
-            </label>
-            <input
-              type='password'
-              id='password'
-              value={enteredPassword}
-              onChange={handlePasswordChange}
-              onBlur={handleOnBlurPassword}
-            />
-          </div>
-          <div className={styles['form-group']}>
-            <button className={styles.btn} type='submit'>
-              Login
-            </button>
+    <div className={styles.container}>
+      <h1>Login</h1>
+      <form className={styles['form']} onSubmit={formSubmissionHandler}>
+        <div
+          className={
+            enteredEmailIsValid
```

Comment:
No need to repeat styles['form-group'] twice, you can do this (same at other two places)
```suggestion
           `${styles['form-group']} ${enteredEmailIsValid ? '' : styles['invalid']}`
```